Packs/OpenAI/Integrations/OpenAiChatGPTV3/OpenAiChatGPTV3.yml

```diff
@@ -0,0 +1,48 @@
+category: Messaging and Conferencing
+commonfields:
+  id: OpenAi ChatGPT v3
+  version: -1
+configuration:
+- defaultvalue: https://api.openai.com/v1/chat/completions
+  display: 'Server URL: (e.g. https://api.openai.com/v1/chat/completions)'
```

Comment:
We use the full URL as the server url, and add the endpoint (v1/chat/completions) to the code. 

```suggestion
- defaultvalue: https://api.openai.com/
  display: 'Server URL: (e.g. https://api.openai.com/)'
```